core/client/fs/src/test/java/alluxio/client/block/policy/LocalFirstPolicyTest.java

```diff
@@ -37,9 +37,9 @@
  */
 public final class LocalFirstPolicyTest {
 
-  private static final InstancedConfiguration S_CONF = ConfigurationTestUtils.copyDefaults();
+  private static final AlluxioConfiguration S_CONF = Configuration.global();
```

Comment:
Maybe for a later change. I don't think `LocalFirstPolicy` needs configuration as a parameter. If you remove that, you can remove this as well. ;)